src/status_im/ui/screens/extensions/add/views.cljs

```diff
@@ -80,6 +80,12 @@
          [react/text (i18n/label :t/invalid-extension)]
          [react/text (str errors)]]]])))
 
+(views/defview show-extension []
+  [show-extension-base false])
```

Comment:
Maybe use a map here? `{:modal? false}` reads a bit better

Reply:
it's just general approach, we use it the same in other modules

Reply:
Ok!